### ISSUE TITLE
skydoc.bazel.build shouldn't recommend depending on sass and node

### ISSUE THREAD

**Issue by brandjon** [bazelbuild/stardoc#11]
The [getting started page](https://skydoc.bazel.build/docs/getting_started_stardoc.html) recommends a WORKSPACE incantation that is no longer needed for Stardoc.

@aiuto

**Comment by c-parsons** [comment]
The plan is to completely take down the skydoc.bazel.build site (it should forward to in-github documentation of this repository).

I'm going to change the issue description to track this here.

**Comment by jin** [comment]
I would reconsider taking down the website, but to serve a (loud) notice to direct Stardoc and new users to this repo.

Taking down the website will break the use case of our existing Skydoc users, which is not ideal.

**Comment by c-parsons** [comment]
Fair enough -- we can leave the **Skydoc** (python binary) portion of the site intact, with strong deprecation warnings, but take down all Stardoc-related information (with a redirection to this repo)

**Comment by jin** [comment]
Sounds good.

**Comment by brandjon** [comment]
Can this issue be closed? Looks like skydoc.bazel.build now makes clear that Stardoc is its replacement.

**Comment by aiuto** [comment]
Related questions:
- Do we want to have subdomains of bazel.build in general
- What is the criteria for having a subdomain page? Do Google projects get privilege?
- Could we host these as github.io pages?

**Comment by brandjon** [comment]
See also [this plan](https://docs.google.com/document/d/1HrUHFyfU0HpWG6F6ws-8zESZ16h4N7p8WsOPR9iqJbo/edit#heading=h.5mcn15i0e1ch) for migrating from the name "skydoc" to "stardoc". We should assess what steps remain as part of this bug.
